tests/unittests/test_client.py

```diff
@@ -28,15 +30,16 @@ def test_access_token_empty_expires(self, mocked_post):
 
         client.fetch_and_set_access_token()
         expires = client.get_expires_time_for_test()
+
         self.assertEqual(expires, datetime.fromtimestamp(future_time))
 
-    def test_access_token_expires_valid(self, mocked_post):
+    def test_access_token_expires_valid(self, mocked_post, mock_write_token):
         '''
         Ensure that we check and return on valid self.__expires
         '''
-        client = _client.LinkedinClient('client_id', 'client_secret', 'refresh_token', 'access_token')
+        client = _client.LinkedinClient('client_id', 'client_secret', 'refresh_token', 'access_token', 'config_path')
 
-        future_time = int(datetime.utcnow().timestamp()) + 100
+        future_time = int(datetime.utcnow().timestamp()) + 88400
```

Comment:
if we change the timedelta in fetch_and_set per the above comment, this duration can be reduced accordingly